### ISSUE TITLE
18MEX: Add icon for Guadalajara hex

### ISSUE THREAD

**Issue by perwestling** [tobymao/18xx#7352]
In 18MEX the Guadalajara hex uses a special tile for upgrade to brown. But the hex uses normal tiles earlier in the game.

After playing 18MEX and 1848 I got an idea for an improvement. 

1848 uses a K chip that is used for tile lay of normal yellow tiles in K hexes. Then they can be upgraded to green.

Now 1848 could be handled by adding a label to yellow tiles, but it is known to have caused bugs. And people might get confused when yellow tiles appearing with new labels on the board and not being present in the tile rooster.

So an alternative could be to use a "label chip" like in GMT 1848. This can be put as an non-sticky icon on the tile. To show potential upgrade (next or future). 

So in the 18MEX case this would mean a sticky G icon on the Guadalajara base hex that stays until Guadalajara is upgraded to brown G tile. 

(Implementation would still have to handle this special upgrades, but I believe this would be less confusing.
The wiki page could also explain the special icons.)

**Comment by thepackrat** [comment]
labels don't cause bugs if you remove the labels on upgrade (per 22, NY, 70 etc). 

I'm against having a different way of marking tiles that basically means the same thing. Quite a few games use labels on the map but have some plain track tiles, then graduate to labelled special tiles in later ranks, so I feel it would be better to continue with the same visual language rather than having a second type of symbol presentation to explain.

**Comment by perwestling** [comment]
@thepackrat - There is a difference. By having a label in the hex for a future upgrade it would be confusing for the players as there are no yellow or green tiles with that label. 

Adding a label to show the next upgrade (like adding a G label to green Guadalajara tile) would be an improvement to the current implementation (and to the actual physical game).

So for me it feels as an improvement to have a label that is different in look than the normal label, to show the possibility of future upgrades. So a light colored G maybe? And then my suggestion to do it as an icon as this does not collide with the labels, and each icon anyway need to be explained. But that is just a suggestion.

In other words - I am opposed to adding a standard label to map hex, or to yellow hex, for 18MEX Guadalajara. And you are opposed to adding some other symbol to denote a future upgrade. 

So maybe adding label to green is a good compromise. (As well as some other games, like 18TN, 18AL, that have similar situations.)

**Comment by thepackrat** [comment]
Look at 1822 T hexes or 1870 large cities. The map has labels but plain yellows lay. People don't seem super confused. 

The 48 K chit is a physical implementation of punching the symbol onto a plain tile. 

The plain map symbol G on Guadalajara would be helpful for people casting their eye across the board asking "where does that big city go" when planning, also in yellow. This was reported as a platime confusion in NY where people were trying to figure out which cities upgraded later. 

And 18xx.games simply shows upgrades. That's the big one. People can just ask.

**Comment by perwestling** [comment]
You do not think that people will be more confused if there is a label on plain map hex, and no yellow tiles with that label? Why change what people are used to? A label does mean that there is a specific tile you can upgrade to.

See, it is no problem to come up with an argument. To make it fact, though, you would need to show both alternatives (or three), and ask people what they prefer.

As 18MEX is now it does follow the physical game, but the implementation stops anyone from doing the wrong upgrade to brown, which is an improvement of the physical game.

As we have an improvement, and I am skeptical that your suggestion would not introduce more confusion, I will be content with that. 

Also, I did promise to not do any GUI changes, so I will just give up on this one. And keep on playing with my K chips in 1848.

**Comment by thepackrat** [comment]
Well, people have coped with it in 1870 for many decades, and titles like the popular 1822 also has map labels which only use plain tiles. Some of the 1822 variants have physical label punch tokens to surface labels on those plain yellows ahead of the symbol marked greens.

So, no, I don't think people will be particularly confused, they clearly cope and understand the visual language.
